static/temp/foods.json

```diff
@@ -0,0 +1,250 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "categoryName": "Fruit and Fruit Juices",
+    "categoryIcon": "EggAltOutlinedIcon",
+    "overview": {
+      "info": "This is a cool food",
+      "NutrientScore": ["A", "B", "C", "D"],
+      "climateImpact": {
+        "title": "Climate Impact",
+        "weight": "20kg co2",
+        "weightTitleSubscript": "Per serving",
+        "equivalent": "200kg CO2 is equivalent to",
```

Comment:
The value `200kg CO2` will be taken from `emissions`, it will be dynamic. So we can remove this here.
```text
"equivalentTitle": "is equivalent to"
```

Reply:
done